ingestion-core/src/metadata/_version.py

```diff
@@ -7,5 +7,5 @@
 
 from incremental import Version
 
-__version__ = Version("metadata", 0, 8, 0, dev=4)
+__version__ = Version("metadata", 0, 8, 0, dev=5)
```

Comment:
this should be taken care by the workflow itself

Reply:
I broke a bunch of tests with my changes in this PR -- will attempt to fix this soon

Reply:
never mind, this needs to manually done with latest changes cc @akash-jain-10